opm/simulators/wells/StandardWell_impl.hpp

```diff
@@ -2043,9 +2044,104 @@ namespace Opm
         }
     }
 
+    template <typename TypeTag>
+    void
+    StandardWell<TypeTag>::addWellPressureEquations(PressureMatrix& jacobian,
+                                                    const BVector& weights,
+                                                    const int pressureVarIndex,
+                                                    const bool use_well_weights,
+                                                    const WellState& well_state) const
+    {
```

Comment:
This part need more documentation. The equations for each variant should be in the comments.